_includes/bios-shrinker.html

```diff
@@ -0,0 +1,2 @@
+<label for="file-input">{{ site.data[page.collection].strings.upload-bios }}</label>
+<input id="file-input" type="file" onchange="shrinkBios(this.files[0])">
```

Comment:
Tested and seems good to me except this should have bootstrap classes:

```suggestion
<label for="file-input" class="form-label">{{ site.data[page.collection].strings.upload-bios }}</label>
<input id="file-input" class="form-control mb-2" type="file" onchange="shrinkBios(this.files[0])">
```